src/Microsoft.DocAsCode.Common/SharedObjects/SharedObjectManager.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.Concurrent;
+
+    public class SharedObjectManager<TState, TOperand>
```

Comment:
rename to TEvent

---
In reply to: [120027375](https://github.com/dotnet/docfx/pull/1723#discussion_r120027375) [](ancestors = 120027375)